gravitee-apim-gateway/gravitee-apim-gateway-handlers/gravitee-apim-gateway-handlers-api/src/main/java/io/gravitee/gateway/jupiter/handlers/api/SyncApiReactor.java

```diff
@@ -95,7 +106,9 @@ public SyncApiReactor(
         final PolicyManager policyManager,
         final FlowChainFactory flowChainFactory,
         final GroupLifecycleManager groupLifecycleManager,
-        final Configuration configuration
+        final Configuration configuration,
+        final long pendingRequestsTimeout,
```

Comment:
I'm concerned by the number of arguments we could have by passing each argument we could directly retrieve from the configuration.

Could it be done like it is already done for tracing ? eg:
```java
this.pendingRequestsTimeout = configuration.getProperty(PENDING_REQUESTS_TIMEOUT_PROPERTY, Long.class, 10_000L);
```